.github/workflows/docker.yaml

```diff
@@ -0,0 +1,29 @@
+name: Docker
+
+on:
+  push:
+    branches:
+      - 'main'
+
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Login to DockerHub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+      - name: Build and push evinced
+        uses: docker/build-push-action@v3
+        with:
+          push: true
+          tags: /evince:latest
```

Comment:
will this automatically be prefixed by quicksilverzone given that's the repo it is pushed to?

Reply:
yup, we need the secrets it would be done.